tsconfig.json

```diff
@@ -25,37 +25,46 @@
     // "moduleDetection": "auto",                        /* Control what method is used to detect module-format JS files. */
 
     /* Modules */
-    "module": "commonjs",                                /* Specify what module code is generated. */
+    "module": "commonjs" /* Specify what module code is generated. */,
     // "rootDir": ".",                                   /* Specify the root folder within your source files. */
     // "moduleResolution": "node10",                     /* Specify how TypeScript looks up a file from a given module specifier. */
-    "baseUrl": "./",                                     /* Specify the base directory to resolve non-relative module names. */
-    "paths": {},                                         /* Specify a set of entries that re-map imports to additional lookup locations. */
-    "rootDirs": ["./lib", "./test"],                     /* Allow multiple folders to be treated as one when resolving modules. */
-    "typeRoots": ["./types", "./node_modules/@types"],   /* Specify multiple folders that act like './node_modules/@types'. */
+    "baseUrl": "./" /* Specify the base directory to resolve non-relative module names. */,
+    "paths": {
+      // workaround for: https://github.com/vitest-dev/vitest/issues/4567
+      "rollup/parseAst": ["./node_modules/rollup/dist/parseAst"]
```

Comment:
Prefer to do dependency updates and large formatting updates as a separate PR, as it makes it much easier to see what actually changed (or broke if a bug gets introduced)